lib/grape/middleware/error.rb

```diff
@@ -92,19 +95,24 @@ def error_response(error = {})
         message = error[:message] || options[:default_message]
         headers = { Grape::Http::Headers::CONTENT_TYPE => content_type }
         headers.merge!(error[:headers]) if error[:headers].is_a?(Hash)
-        backtrace = error[:backtrace] || []
-        rack_response(format_message(message, backtrace), status, headers)
+        backtrace = error[:backtrace] || error[:original_exception] && error[:original_exception].backtrace || []
```

Comment:
Grape supports whatever is in https://github.com/ruby-grape/grape/blob/master/.travis.yml, so 2.2+, so we can tame Rubocop accordingly. Since we don't use `&.` anywhere else maybe just expanding it is better.